contents/docs/features/paths.md

```diff
@@ -4,25 +4,35 @@ sidebar: Docs
 showTitle: true
 ---
 
-Paths are PostHog's way of letting you, inspect free form how traffic is flowing through your application. A path can be comprised of any action (such as pageviews, or button clicks, or a combination of both).
+Paths are PostHog's way of letting you inspect free form how traffic is flowing through your application. A path can be comprised of any action (such as pageviews, button clicks, or a combination of both). 
 
 The kind of ways Paths can be helpful are as follows:
 
 * Understand where traffic is landing into your application or website
 * Understand where traffic is quitting your application
 * Understand which parts of your application people are actually using
 
-## Viewing paths
+We provide a visual representation of the paths taken by users on your product, so that you can easily identify these trends.
```

Comment:
'trends' perhaps confusing language-wise given it's a different piece of functionality

Reply:
Right, "patterns" is probably the way to go